assets/src/dashboard/components/navigation-bar.js

```diff
@@ -82,7 +82,7 @@ const LinksContainer = styled.div`
   }
 `;
 
-const NewStoryLink = styled(Button)`
+const NewStoryLink = styled(Button).attrs({ onClick: () => {} })`
```

Comment:
Since this button gets forwarded as a `a` tag, the props still throw an error because there is a missing `onClick` prop. This fixes that warning.

<img width="790" alt="Screen Shot 2020-03-30 at 11 59 20 AM" src="https://user-images.githubusercontent.com/1738349/77940953-0c3da100-727f-11ea-8ce7-35cdb9540d31.png">

Reply:
Would it be useful to make an anchor version of the button? I can do that and then sub this out after tackling search.